### ISSUE TITLE
Support rotating through dashboards

### ISSUE THREAD

**Issue by davejlong** [kittoframework/kitto#29]
Often times dashboards are shown on a TV and sometimes you can't show all the information that needs to be shown on a single page. It'd be nice to have support for building a dashboard that rotates between a list of other dashboards with a time between rotation. The dashboard template would be something like:

```
<div data-rotate="sample,jobs,elixir" data-every="60"></div>
```

This would trigger a rotation between the sample, jobs and elixir dashboards every 60 seconds.

**Comment by zorbash** [comment]
I like the idea. I can see some issues with the proposed API though.

If i append `<div data-rotate="sample,jobs,elixir" data-every="60"></div>` to one of my dashboards one won't be able to opt-out of dashboards rotation.

Alternatively we can use somethings like

`GET /dashboards?rotate=sample,jobs&every=60` to achieve what this issue suggests.

What do you think?

**Comment by JanStevens** [comment]
We have used the following in the original dashing: https://gist.github.com/jwalton/6670630
It allowed to stop or only show a specific dashboard, maybe we can convert this to react?

**Comment by zorbash** [comment]
@JanStevens we can do something like https://github.com/vrish88/sinatra_cyclist mentioned in the gist you posted. I think should add a minimal amount of js / scss for this feature. I don't think having fancy transitions is required for most people.

**Comment by JanStevens** [comment]
@Zorbash The advantage of the JS version is that its a virtual split up and not backend side.
We used it in combination with dynamic dashboard (widget configs where defined in a json file and then rendered).

You can have one gigantic dashboard (url) split up every 12 widgets in a virtual different page. You can add transitions (yes customers love transitions), force to only show one page, timeouts,...

**Comment by davejlong** [comment]
(Warning: unvalidated statistics ahead) One thing to keep in mind as we
build the JS framework for Kitto over time is that dashboards are generally
displayed using a relatively low powered computer with low browser
performance. Adding more processing, especially GPU intensive processing
will kill performance.

**Comment by JanStevens** [comment]
@davejlong we have run original dashing dashboards, with that JS rotating plugin and 24 widgets (counters / graphs) on a raspberry pi 1 with an updating frequency of 10sec. No issue.

**Comment by zorbash** [comment]
Implemented in #42.
